applications/GeoMechanicsApplication/custom_conditions/U_Pw_face_load_interface_condition.hpp

```diff
@@ -26,83 +26,88 @@
 namespace Kratos
 {
 
-template< unsigned int TDim, unsigned int TNumNodes >
-class KRATOS_API(GEO_MECHANICS_APPLICATION) UPwFaceLoadInterfaceCondition : public UPwCondition<TDim,TNumNodes>
+template <unsigned int TDim, unsigned int TNumNodes>
+class KRATOS_API(GEO_MECHANICS_APPLICATION) UPwFaceLoadInterfaceCondition
+    : public UPwCondition<TDim, TNumNodes>
 {
-
 public:
+    KRATOS_CLASS_INTRUSIVE_POINTER_DEFINITION(UPwFaceLoadInterfaceCondition);
 
-    KRATOS_CLASS_INTRUSIVE_POINTER_DEFINITION( UPwFaceLoadInterfaceCondition );
-
-    using IndexType = std::size_t;
+    using IndexType      = std::size_t;
     using PropertiesType = Properties;
-    using NodeType = Node;
-    using GeometryType = Geometry<NodeType>;
+    using NodeType       = Node;
+    using GeometryType   = Geometry<NodeType>;
     using NodesArrayType = GeometryType::PointsArrayType;
-    using VectorType = Vector;
-    using MatrixType = Matrix;
+    using VectorType     = Vector;
+    using MatrixType     = Matrix;
 
-//----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+    //----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 
     UPwFaceLoadInterfaceCondition() : UPwFaceLoadInterfaceCondition(0, nullptr, nullptr) {}
 
-    UPwFaceLoadInterfaceCondition( IndexType               NewId,
-                                   GeometryType::Pointer   pGeometry )
+    UPwFaceLoadInterfaceCondition(IndexType NewId, GeometryType::Pointer pGeometry)
         : UPwFaceLoadInterfaceCondition(NewId, pGeometry, nullptr)
-    {}
+    {
+    }
 
-    UPwFaceLoadInterfaceCondition( IndexType               NewId,
-                                   GeometryType::Pointer   pGeometry,
-                                   PropertiesType::Pointer pProperties )
-        : UPwCondition<TDim,TNumNodes>(NewId, pGeometry, pProperties)
+    UPwFaceLoadInterfaceCondition(IndexType NewId, GeometryType::Pointer pGeometry, PropertiesType::Pointer pProperties)
+        : UPwCondition<TDim, TNumNodes>(NewId, pGeometry, pProperties)
     {
         // Lobatto integration method with the integration points located at the "mid plane nodes" of the interface
         this->SetIntegrationMethod(GeometryData::IntegrationMethod::GI_GAUSS_1);
     }
 
-//----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+    //----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 
```

Comment:
There are  many such lines. Should it be a separate issue to remove them?

Reply:
Yes, I will create a separate issue on the product backlog to remove these redundant separator lines.